src/main/java/org/elasticsearch/index/query/MatchAllQueryBuilder.java

```diff
@@ -64,15 +56,19 @@ public MatchAllQueryBuilder boost(float boost) {
         return this;
     }
 
+    /**
+     * @return the boost for this query.
+     */
+    public float getBoost() {
```

Comment:
I think getters don't hurt and should have already been there. At the moment stuff can't be set but not retrieved, which seems silly. If we want to keep bw comp then let's leave the setter alone and follow the same convention with getters too